docs/source/azure/policy/genericarmfilter.rst

```diff
@@ -3,3 +3,57 @@
 Filters for ARM Resources
 =========================
 
+``MetricFilter``
+Filters Azure resources based on live metrics from the Azure monitor.
+
+.. c7n-schema:: MetricFilter
+    :module: c7n_azure.filters
+
+
+Example Policies
+----------------
+
+Find VMs with an average Percentage CPU greater than or equal to 75% over the last 12 hours
+
+.. code-block:: yaml
+
+    policies:
+      - name: find-busy-vms
+        resource: azure.vm
+        filters:
+          - type: metric
+            metric: Percentage CPU
+            aggregation: average
+            op: ge
+            threshold: 75
+            timeframe: 12
+
+Find KeyVaults with more than 1000 API hits in the last hour
+
+.. code-block:: yaml
+
+    policies:
+      - name: keyvault-hits
+        resource: azure.keyvault
+        filters:
+        - type: metric
```

Comment:
minor: please indent this filter block in one level as well as the next policy `dtu-consumption` for consistency